docs/gno-tooling/cli/gnodev.md

```diff
@@ -121,3 +121,4 @@ While `gnodev` is running, the following shortcuts are available:
 | --server-mode       | disable interaction, and adjust logging for server use.    |
 | --verbose           | enable verbose output for development                      |
 | --web-listener      | web server listening address                               |
+| --web-help-remote   | web server help page's remote addr                         |
```

Comment:
Eventually, we can use `--web-` or `--gnoweb-` as prefixes for Gnoweb, and `--node-` or `--gnoland-` for the node.

Reply:
Agree. For now, `web-help-remote` is the only gnoweb option that makes sense for gnodev. Later we can add others like `web-captcha-site` if needed.